feat: Add Badge component (#1045)
Amsterdam/design-system: packages/css/src/components/badge/README.md

# Badge

A prominently colored box containing 1 or 2 words to guide the user in taking a specific action.

## Design

> The badge can contain a string of text or a number.
> 
> ## Guidelines
> 
> The default background colour is green.
> Suggestions when to use the other colours will follow soon.

Reviewer: These are not guidelines, so let’s merge the sentences into the ‘Design’ section.

```suggestion
The badge can contain a short text or a number.
The default background colour is dark green.
Suggestions on when to use the other colours will follow soon.
```

# Badge

> A prominently colored box containing 1 or 2 words to guide the user in taking a specific action.

Reviewer: ```suggestion
A prominently coloured box containing 1 or 2 words.
Guides the user in taking a specific action or describes its surrounding content.
```
